src/components/SignUpForm/index.jsx

```diff
@@ -15,13 +15,13 @@ const SignUpForm = () => {
       <div className="sign-up-form__title">
         <h1>SignUp</h1>
       </div>
-      <div className="sign-up-form__social-login">
+      <div className="col-2">
```

Comment:
let stick to bem methodology here and scope this too

Reply:
Done.